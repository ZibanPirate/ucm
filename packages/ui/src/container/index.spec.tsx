import { render } from "@testing-library/react";

import { Container } from ".";

describe(`Testing component '${Container.name}' :`, () => {
  it(`should render with 4 div elements`, () => {
    const { container } = render(
      <Container>
        {[1, 2, 3, 4].map((index) => (
          <div key={index}>{index}</div>
        ))}
      </Container>,
    );
    expect(container).toMatchSnapshot();
  });
});
