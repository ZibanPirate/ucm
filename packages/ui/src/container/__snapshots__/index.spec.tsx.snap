// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component 'Container' : should render with 4 div elements 1`] = `
<div>
  <div
    style="max-width: 900px; margin: auto;"
  >
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
    <div>
      4
    </div>
  </div>
</div>
`;
